src/explorer.api/ExplorerConfig.cs

```diff
@@ -1,13 +1,36 @@
 namespace Explorer.Api
 {
     using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
 
-    public class ExplorerConfig
+    using Aircloak.JsonApi;
+    using Explorer.Components;
+    using Explorer.Metrics;
+
+    public class ExplorerConfig : IAircloakAuthenticationProvider, PublisherComponent
     {
-        public const string ApiKeyEnvironmentVariable = "AIRCLOAK_API_KEY";
+        public string AircloakApiKey { get; set; } = string.Empty;
 
         public uint PollFrequency { get; set; } = 2000;
 
         public TimeSpan PollFrequencyTimeSpan => TimeSpan.FromMilliseconds(PollFrequency);
+
+        public string CommitHash { get; set; } = string.Empty;
+
+        public string CommitRef { get; set; } = string.Empty;
+
+        public Task<string> GetAuthToken() => Task.FromResult(AircloakApiKey);
+
+        public async IAsyncEnumerable<ExploreMetric> YieldMetrics()
+        {
+            yield return new UntypedMetric(
+                name: "version_info",
+                metric: new
+                {
+                    CommitHash,
+                    CommitRef,
+                });
+        }
```

Comment:
I would prefer to somehow publish the version info outside the column metrics (i.e. at the same level with the exploration id). With the current implementation the `version_info` is unnecessarily repeated for each column. Also when just looking at an explorer result, I will rather have this information at the beginning, so I will not have to search it in the JSON.

Reply:
Ok I think I agree. In that case the information will need to be added to the `ExploreResult` class instead I suppose. I'll change this.

Reply:
This is done. 8625dde